packages/sources/wbtc-address-set/src/endpoint/addresses.ts

```diff
@@ -36,7 +36,12 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
 
   const result = response.data.result
     .filter((a) => a.type == 'custodial' && a.balance)
-    .map((a) => ({ ...a, coin: 'btc', chain: 'mainnet' }))
+    .map<Account>((a) => ({
+      ...a,
+      balance: typeof a.balance !== 'undefined' ? String(a.balance) : undefined,
+      coin: 'btc',
+      chain: 'mainnet',
+    }))
```

Comment:
Karen is working on updating these adapters in #1129. Might make sense to leave those changes for that PR if possible

Reply:
ahhh right, did not see that before

Reply:
Rebased and changed target branch (for now).